NeuroID/NeuroIDClass/Extensions/NIDAdvancedDevice.swift

```diff
@@ -80,10 +83,12 @@ public extension NeuroID {
         }
     }
     
-    internal static func captureADVEvent(_ requestID: String, cached: Bool) {
+    internal static func captureADVEvent(_ requestID: String, cached: Bool, latency: Double) {
         let nidEvent = NIDEvent(type: .advancedDevice)
         nidEvent.rid = requestID
         nidEvent.c = cached
+        nidEvent.l = latency
+        nidEvent.ct = NetworkMonitoringService.connectionTypeAccessor.rawValue
```

Comment:
Note:
We have a variable on the NeuroID class `networkMonitor` that is an instance of the `NetworkMonitoringService` class, is there a reason we wouldn't use that with the `connectionType` attribute instead?